colors/nord.vim

```diff
@@ -168,7 +168,7 @@ call s:hi("Underline", "", "", "", "", s:underline, "")
 call s:hi("ColorColumn", "", s:nord1_gui, "NONE", s:nord1_term, "", "")
 call s:hi("Cursor", s:nord0_gui, s:nord4_gui, "", "NONE", "", "")
 call s:hi("CursorLine", "", s:nord1_gui, "NONE", s:nord1_term, "NONE", "")
-call s:hi("Error", s:nord0_gui, s:nord11_gui, "", s:nord11_term, "", "")
+call s:hi("Error", s:nord4_gui, s:nord11_gui, "", s:nord11_term, "", "")
```

Comment:
Could you please post comparison screenshots between Vim in _terminal_ and _GUI_ mode showing the current differences?

Reply:
This is the comparison of highlighting for the `Error` group between console vim (set notermguicolors) and gvim prior to this PR:
gvim:
![Screenshot from 2020-04-28 09:02:11](https://user-images.githubusercontent.com/16892446/80491288-815cce80-8930-11ea-89e8-2f7ea4dd6271.png)
vim:
![Screenshot from 2020-04-28 09:02:48](https://user-images.githubusercontent.com/16892446/80491319-8d489080-8930-11ea-8444-d42e28af15cb.png)

This patch changes the highlighting for `gvim` to match that of console `vim`:
gvim after the patch:
![Screenshot from 2020-04-28 09:19:28](https://user-images.githubusercontent.com/16892446/80491987-6ccd0600-8931-11ea-9a0e-9a98a14e8fef.png)

Reply:
The darker font on red is a bad contrast and definitely not the way it should look like so this is a required change 👍